src/main/java/org/openelisglobal/sample/form/ProjectData.java

```diff
@@ -485,13 +485,6 @@ public void setEIDSites(List eIDSites) {
         EIDSites = eIDSites;
     }
 
-    public String getEIDsiteName() {
```

Comment:
@manishjha-04 wont removing these properties brak the OLD ui ??

Reply:
Hey @mozzy11 , I've verified, and the update doesn't affect the old UI.
 Additionally, the properties I removed were duplicates. You can check in the codebase to see these duplications.